packages/plugin-session-replay-browser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amplitude/plugin-session-replay-browser",
-  "version": "0.11.0-beta.0",
+  "version": "1.0.0-beta.0",
```

Comment:
Are you keeping this in beta? I think adding the dependency at `1.0.0` will not retrieve a beta version.

Reply:
Ah okay, so 1.0.0 is fine? I was going to pre-release then graduate but okay with 1.0.0 if that is correct procedure.

Reply:
I think its ok to do a prelease and then graduate. Just wasn't sure if you had kept beta intentionally

Reply:
Okay thanks! I changed to 1.0.0 anyways since we would graduate immediately